lhotse/recipes/timit.py

```diff
@@ -71,8 +72,17 @@ def prepare_timit(
 
     manifests = defaultdict(dict)
     dataset_parts = ['TRAIN', 'DEV', 'TEST']
+    
+    phones_dict = {}
 
-    phones_dict = get_phonemes(num_phones)
+    try:
```

Comment:
Is there a need to use try .. except here?
What happens if it is removed?

Reply:
Em....it is to remind people to set num_phones in [60, 48, 39]. If the value of num_phones not in [60, 48, 39], the error happens.

Reply:
I agree with @csukuangfj that this try/except block is unnecessary. Please change:

```python
    try:
        if num_phones in [60, 48, 39]:
             phones_dict = get_phonemes(num_phones)
         else:
             raise ValueError("The value of num_phones must be in [60, 48, 39].")
     except ValueError as e:
         print("Exception: ", repr(e))
         raise 
```

into

```python
        if num_phones in [60, 48, 39]:
             phones_dict = get_phonemes(num_phones)
         else:
             raise ValueError("The value of num_phones must be in [60, 48, 39].")
```